apps/condo/domains/common/components/Import/Index.tsx

```diff
@@ -31,6 +31,7 @@ interface IImportWrapperProps {
     accessCheck: boolean
     onFinish(): void
     columns: Columns
+    maxTableLength: number
```

Comment:
```suggestion
    maxTableLength?: number
```
Если это сделать опциональным и ниже указать дефолтное значение, тогда не придется хардкодить число `500` на страницах домов и контактов